src/lib/gql/__generated__/drupal.d.ts

```diff
@@ -2512,6 +2782,70 @@ export type QueryTermArgs = {
   langcode?: InputMaybe<Scalars['String']['input']>;
 };
 
+/** Entity type redirect. */
```

Comment:
Could this have something to do with why redirects don't seem to take effect correctly when made on the Drupal side?